### DISCUSSION TITLE
Smooth color picker

### DISCUSSION THREAD

**Discussion by Flashwalker** [facelessuser/ColorHelper#244] [Q&A] [answered]
Is there a way to have a smooth color gradient in the color picker instead of discrete colors?

**Comment by facelessuser** [comment, marked as answer]
We are limited by what Sublime's minihtml can do. You have the color box and slider, if you click the slider label, you'll get even better resolution. It's really the best that could be done in the limited HTML. You can always go to tool `edit and mix` and just adjust the numbers directly. It has live previews.

**Reply by Flashwalker** [reply]
ok, thanks
